public/assets/css/main.css

```diff
@@ -78,6 +78,31 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
+	list-style-type: none;
+	font-size: 80%;
+}
+
+ul li {
+	margin: 1em;
+}
+
+footer {
```

Comment:
text-align: center; to center the text

Reply:
Footer text set to `center` using other property.